extensions/package-vulnerability-scanner/src/components/ContentCard.vue

```diff
@@ -114,14 +114,16 @@ function handleClick() {
     </div>
 
     <div class="mt-2 text-sm">
-      <span v-if="packageCount > 0"> {{ packageCount }} packages </span>
-      <span v-else-if="hasError" class="text-red-600">
+      <span v-if="packageCount > 0" class="text-gray-600">
+        {{ packageCount }} packages
+      </span>
```

Comment:
Here is an example of an unstyled block that is now using a `text-gray-` specification.